$NetBSD$

Fixed location of xbps.d.5 manpage.
Pull-request is created on github.

--- data/Makefile.orig	2018-07-30 11:47:08.000000000 +0000
+++ data/Makefile
@@ -11,8 +11,8 @@ all:
 
 .PHONY: install
 install:
-	install -d $(DESTDIR)$(SHAREDIR)/man/man5
-	install -m644 xbps.d.5 $(DESTDIR)$(SHAREDIR)/man/man5
+	install -d $(DESTDIR)$(MANDIR)/man5
+	install -m644 xbps.d.5 $(DESTDIR)$(MANDIR)/man5
 	install -d $(DESTDIR)$(PKGCONFIGDIR)
 	install -m644 $(PCFILE) $(DESTDIR)$(PKGCONFIGDIR)
 	install -d $(DESTDIR)/$(DBDIR)/keys
